packages/core/src/comments/extension.ts

```
@@ -223,8 +225,9 @@ export const CommentsExtension = createExtension(
               );
             },
             handleClick: (view, pos, event) => {
+              console.log("comments");
```

ESLint

[error] Unexpected console statement.

(no-console)